static/css/gridpage.css

```diff
@@ -316,11 +316,6 @@ section.bullets .content {
 }
 
 @media screen and (max-width: 480px){
-    .gridPage #hero {
-        padding-right: 20px;
```

Comment:
Hello @Arhell ,
Just looked at the changes.
Are the `.gridPage` changes essential?

Reply:
I returned the changes, it seemed to me that it looks better